### ISSUE TITLE
QA: dashboard smoke test for {octopus,ses7}

### ISSUE THREAD

**Issue by smithfarm** [SUSE/sesdev#292]
After deployment, the URL of the running dashboard instance can be introspected by:

```
DASHBOARD_URL="$(ceph mgr services | jq -r .dashboard)"
```

and then one can do something like

```
set -e
curl --silent --insecure "$DASHBOARD_URL" | grep -i "SUSE Enterprise Storage"
```

**Comment by smithfarm** [comment]
This is doable now for {ses5,nautilus,ses6}, but {octopus,ses7,pacific} is blocked by https://bugzilla.suse.com/show_bug.cgi?id=1170498

**Comment by smithfarm** [comment]
Fixed by #392
